core/pom.xml

```diff
@@ -28,6 +28,11 @@
       <version>0.11.0</version>
       <scope>provided</scope>
     </dependency>
+    <dependency>
+      <groupId>com.google.guava</groupId>
+      <artifactId>guava</artifactId>
+      <version>18.0</version>
```

Comment:
If we get it transitively it's not a huge issue I suppose

Reply:
No, but rather avoid surprises for now - plus I think guava comes in transitively at `provided` currently